packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -19,18 +19,30 @@ import {
 } from "@fluidframework/driver-utils";
 import { ChildLogger } from "@fluidframework/telemetry-utils";
 import { DocumentService } from "./documentService";
+import { IRouterliciousDriverPolicies } from "./policies";
 import { ITokenProvider } from "./tokens";
 import { RouterliciousOrdererRestWrapper } from "./restWrapper";
 
+const defaultRouterliciousDriverPolicies: IRouterliciousDriverPolicies = {
+    enablePrefetch: true,
+};
+
 /**
  * Factory for creating the routerlicious document service. Use this if you want to
  * use the routerlicious implementation.
  */
 export class RouterliciousDocumentServiceFactory implements IDocumentServiceFactory {
     public readonly protocolName = "fluid:";
+    private readonly driverPolicies: IRouterliciousDriverPolicies;
+
     constructor(
         private readonly tokenProvider: ITokenProvider,
+        driverPolicies: Partial<IRouterliciousDriverPolicies> = {},
     ) {
+        this.driverPolicies = {
+            ...defaultRouterliciousDriverPolicies,
+            ...driverPolicies,
+        };
```

Comment:
Given that the only property in there is required (not optional), do we need defaults?
Note that in ODSP we make every option optional, such that additions of new policy is always non-breaking change and that driver has some reasonable defaults.

Reply:
The parameter into DocumentServiceFactory is `Partial<...>` which makes all the properties optional, thus requiring defaults. I think I read somewhere that it's better to use Partial<> than manually making each interface property optional, but I have no idea where or if it was a reputable source. I'm open to making each property optional within the interface if that's a better practice. Let me know either way :)